helm/prometheus-rules/templates/alerting-rules/inhibit.prometheus-agent.rules.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: inhibit.prometheus-agent.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: inhibit.prometheus-agent
+    rules:
+    - alert: InhibitionClusterIsNotRunningPrometheusAgent
+      annotations:
+        description: '{{`Cluster ({{ $labels.cluster_id }}) is not running Prometheus Agent.`}}'
+      expr: count(label_replace(kube_namespace_created{namespace=~".+-prometheus"}, "cluster_id", "$1", "namespace", "(.+)-prometheus") ) by (cluster_id) unless count(label_replace(app_operator_app_info{app="observability-bundle",status="deployed",name=~".+-observability-bundle"},"cluster_id", "$1", "name", "(.+)-observability-bundle")) by (cluster_id)
```

Comment:
TBH that looks really cool :) 

Can you add a comment explaining what this does?

@hervenicol do you mind taking a look? You master promql more than me :)